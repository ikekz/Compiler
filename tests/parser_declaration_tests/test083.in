type
	t1 = record
		x: double;
		y: array of integer;
		end;
	t2 = record
			x, y: t1;
		end;
begin
end.