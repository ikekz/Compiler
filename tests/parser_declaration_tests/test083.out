integer        type           integer
double         type           double
char           type           char
t1             type           record:
                              x              recordfield    double
                              y              recordfield    array of integer
                              end
t2             type           record:
                              x              recordfield    record:
                                                            x              recordfield    double
                                                            y              recordfield    array of integer
                                                            end
                              y              recordfield    record:
                                                            x              recordfield    double
                                                            y              recordfield    array of integer
                                                            end
                              end
